gradle/verification-metadata.xml

```diff
@@ -637,6 +637,11 @@
             <pgp value="31bae2e51d95e0f8ad9b7bcc40a3c4432bd7308c"/>
          </artifact>
       </component>
+      <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.15">
+         <artifact name="gradle-enterprise-gradle-plugin-3.15.jar">
+            <pgp value="0ef2cac6b3577539b7ab5bcc91aa2c186c4d7a86"/>
+         </artifact>
+      </component>
```

Comment:
❌ Instead of doing this, please add the new key above as trusted key for all the Gradle artifacts: https://github.com/gradle/gradle/blob/c178d771110b14fcbfc4f8d322cfcb405e14abd6/gradle/verification-metadata.xml#L88-L97

My understanding is that the new key will be used to sign all our artifacts and replaces the old key. Maybe we could move the things that are already signed with the new key under the other trusted entry. We should probably start out at least with
```
<trusting group="com.gradle"/>
```

Reply:
Done in 3fe2b627424a824da2a52c29ddfd699093cb6a9e